### PR TITLE
Layer iconography

### PR DESCRIPTION
**Pull request by ricardoduplos**

### CODE REVIEW

**Comment by olafveerman** [comment]
This looks great.

@ricardoduplos One thing I noticed: Mean Wind Speed is the last layer in the panel. On my screen it sits on the bottom of the visible area. When I click the info icon, nothing seems to happen, while in reality the info drawer opens outside of the visible area. My first thought was that no additional information was available.

## Docs
I will add documentation to this PR, so let's hold off of merging for now.

**Comment by ricardoduplos** [comment]
@olafveerman Thanks for the feedback. I'm aware of the drawer visibility problem. I'm discussing possible solutions with @danielfdsilva.

**Comment by olafveerman** [comment]
The latest commits add:

* documentation for legends
* basic validation for the legend definition 

Both need feedback from @danielfdsilva for accuracy and completeness.

## :bug: 
@danielfdsilva I noticed a bug, when `dashed` is not specified on a `line` layer, the app breaks:

```
There was an error in your GraphQL query:
Cannot query field "dashed" on type "LayerLegend".
```

File: gatsby-node.js [hunk 1]
```diff
@@ -6,6 +6,15 @@ exports.createSchemaCustomization = ({ actions, schema }) => {
 
   const typeDefs = [
     `
+      type LayerLegend {
+        type: String
+        min: String
+        max: String
+        stops: [String]
+        color: String
+        icon: String
```
Comment (danielfdsilva) [inline gatsby-node.js:15]:
```suggestion
        icon: String
        dashed: Boolean
```

**Review by danielfdsilva — CHANGES_REQUESTED** [review]

**Review by olafveerman — APPROVED** [review]

**Pull request merged**